docs/source/jupyter/sdk.ipynb

```diff
@@ -101,18 +126,17 @@
     }
    ],
    "source": [
-    "config = {'configuration': 1}\n",
-    "transformation_id = mp.new_transformation(config)\n",
+    "from marketplace_standard_app_api.models.transformation import NewTransformationModel, TransformationId\r\n",
+    "\r\n",
+    "transformation_id: TransformationId = mp.new_transformation(NewTransformationModel(parameters={\"some_parameter\": \"some_value\"})).id\r\n",
```

Comment:
I'm still wondering whether this can be shortened to
```suggestion
    "transformation_id: TransformationId = mp.new_transformation(parameters={\"some_parameter\": \"some_value\"}).id\r\n",
```
Can we test that?